corehq/motech/repeaters/management/commands/find_missing_repeat_records.py

```diff
@@ -193,16 +192,16 @@ def find_missing_case_repeat_records_for_domain(domain, startdate, enddate, shou
 def find_missing_case_repeat_records_for_case(case, domain, repeaters, startdate, enddate, should_create=False):
     successful_count = missing_all_count = missing_create_count = missing_update_count = 0
 
-    repeat_records = get_repeat_records_by_payload_id(domain, case.get_id)
+    repeat_records = SQLRepeatRecord.objects.filter(domain=domain, payload_id=case.get_id)
     # grab repeat records that were registered during the date range
     records_during_daterange = [record for record in repeat_records
-                                if startdate <= record.registered_on.date() <= enddate]
+                                if startdate <= record.registered_at.date() <= enddate]
```

Comment:
I made minimal changes in this management command to hopefully keep it working on SQL. Two separate queries are likely less efficient than one, but I do think the query could be made slightly more efficient by excluding records registered before the date range since they are unused.
```py
repeat_records = SQLRepeatRecord.objects.filter(..., registered_at__gte=startdate)
```
I don't know how often this command is run, but my guess is not often. That kind of optimization feels strictly outside the scope of this migration.